SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,
+);
+
+CREATE TABLE Tickets
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price INT,
+Validity VARCHAR(150),

```

Comment:
For Validity a new table, maybe called, ValidityTypes could be used. Inside ValidityTypes we can store information like "one day", "one month", "one week" etc. and then it can be referenced in the Tickets table.